exchanges/bybit/bybit_test.go

```diff
@@ -0,0 +1,3302 @@
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestChangeMode(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() || !canManipulateRealOrders {
+		t.Skip("skipping test: api keys not set or canManipulateRealOrders set to false")
+	}
+	pair, err := currency.NewPairFromString("BTCUSDZ22")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	_, err = b.ChangeMode(context.Background(), pair, "Partial")
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestChangeMargin(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() || !canManipulateRealOrders {
+		t.Skip("skipping test: api keys not set or canManipulateRealOrders set to false")
+	}
+	pair, err := currency.NewPairFromString("BTCUSDZ22")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	err = b.ChangeMargin(context.Background(), pair, 1, 1, false)
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestGetTradeRecords(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip("skipping test: api keys not set")
+	}
+	pair, err := currency.NewPairFromString("BTCUSDZ22")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	_, err = b.GetTradeRecords(context.Background(), pair, "", "", 0, 0, 0)
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestGetClosedTrades(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip("skipping test: api keys not set")
+	}
+	pair, err := currency.NewPairFromString("BTCUSDZ22")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	_, err = b.GetClosedTrades(context.Background(), pair, "", 0, 0, 0, 0)
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestSetRiskLimit(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() || !canManipulateRealOrders {
+		t.Skip("skipping test: api keys not set or canManipulateRealOrders set to false")
+	}
+	pair, err := currency.NewPairFromString("BTCUSDZ22")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	_, err = b.SetRiskLimit(context.Background(), pair, 2, 0)
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+// Miscellaneous
+
+func TestTimeSecUnmarshalJSON(t *testing.T) {
+	t.Parallel()
+	tInSec := time.Now().Unix()
+
+	var ts bybitTimeSec
+	err := ts.UnmarshalJSON([]byte(strconv.Itoa(int(tInSec))))
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if !time.Unix(tInSec, 0).Equal(ts.Time()) {
+		t.Errorf("TestTimeSecUnmarshalJSON failed")
+	}
+}
+
+func TestTimeMilliSecUnmarshalJSON(t *testing.T) {
+	t.Parallel()
+	tInMilliSec := time.Now().UnixMilli()
+
+	var tms bybitTimeMilliSec
+	err := tms.UnmarshalJSON([]byte(strconv.Itoa(int(tInMilliSec))))
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if !time.UnixMilli(tInMilliSec).Equal(tms.Time()) {
+		t.Errorf("TestTimeMilliSecUnmarshalJSON failed")
+	}
+}
+
+func TestTimeNanoSecUnmarshalJSON(t *testing.T) {
+	t.Parallel()
+	tInNanoSec := time.Now().UnixNano()
+
+	var tns bybitTimeNanoSec
+	err := tns.UnmarshalJSON([]byte(strconv.Itoa(int(tInNanoSec))))
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if !time.Unix(0, tInNanoSec).Equal(tns.Time()) {
+		t.Errorf("TestTimeNanoSecUnmarshalJSON failed")
+	}
+}
```

Comment:
This is a minor fix, but these tests are impacting 32bit checks. As you can see [here](https://github.com/thrasher-corp/gocryptotrader/runs/7338297939?check_suite_focus=true#step:9:475)
Its from the unixMilli and unixNano being too large.
Changing it to this fixed the issue:

```suggestion
func TestTimeMilliSecUnmarshalJSON(t *testing.T) {
	t.Parallel()
	tInMilliSec := time.Now().UnixMilli()

	var tms bybitTimeMilliSec
	err := tms.UnmarshalJSON([]byte(strconv.FormatInt(tInMilliSec, 10)))
	if err != nil {
		t.Fatal(err)
	}

	if !time.UnixMilli(tInMilliSec).Equal(tms.Time()) {
		t.Errorf("TestTimeMilliSecUnmarshalJSON failed")
	}
}

func TestTimeNanoSecUnmarshalJSON(t *testing.T) {
	t.Parallel()
	tInNanoSec := time.Now().UnixNano()

	var tns bybitTimeNanoSec
	err := tns.UnmarshalJSON([]byte(strconv.FormatInt(tInNanoSec, 10)))
	if err != nil {
		t.Fatal(err)
	}

	if !time.Unix(0, tInNanoSec).Equal(tns.Time()) {
		t.Errorf("TestTimeNanoSecUnmarshalJSON failed")
	}
}
```